client/src/templates/Challenges/projects/ProjectForm.js

```diff
@@ -26,15 +26,6 @@ export class ProjectForm extends Component {
     super(props);
     this.handleSubmit = this.handleSubmit.bind(this);
   }
-  componentDidMount() {
```

Comment:
maybe we leave this one just in case?

Reply:
Yes, I agree. Better to leave it just in case.